var/spack/repos/builtin/packages/soapdenovo-trans/package.py

```diff
@@ -16,13 +16,18 @@ class SoapdenovoTrans(MakefilePackage):
 
     version('1.0.4', sha256='378a54cde0ebe240fb515ba67197c053cf95393645c1ae1399b3a611be2a9795')
 
+    depends_on('zlib', type='link', when='target=aarch64:')
+    depends_on('samtools@0.1.8', type='link', when='target=aarch64:')
```

Comment:
These dependencies should probably exist for all systems. We don't want packages using pre-built vendored dependencies.

Reply:
Delete the condition for aarch64.